python/ray/workers/setup_runtime_env.py

```diff
@@ -51,7 +51,7 @@ def setup(input_args):
                                       sys.version_info[:3]))  # like 3.6.10
             ray_pip = current_ray_pip_specifier()
             if ray_pip and not runtime_env.get("_skip_inject_ray"):
-                extra_pip_dependencies = [ray_pip]
+                extra_pip_dependencies = [ray_pip, "ray[default]"]
             else:
```

Comment:
Does this also work for nightlies

Reply:
Yes, current_ray_pip_specifier will return the appropriate wheel URL for nightlies as well as for the release branch.